src/packageSourceGenerator/PackageSourceGenerator.proj

```diff
@@ -295,6 +295,7 @@
                    IncludeTargetFrameworks="$(PackageTargetFrameworks)"
                    ExcludeTargetFrameworks="$(ExcludeTargetFrameworks)"
                    RemoveIcon="true"
+                   RemoveReadMe="true"
```

Comment:
Kept the implementation the same as before, just added one more option
Not sure if we want to simplify this somehow or maybe its not worth it